### PR TITLE
Issue 515  pytest 30 take 2

### PR DESCRIPTION
**Pull request by adler-j**
New PR instead of #516.

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -67,11 +79,9 @@ def fn_impl(request):
 @pytest.fixture(scope="module", ids=ntuples_impl_ids,
                 params=ntuples_impl_params)
 def ntuples_impl(request):
+    """String with a available `NtuplesBase` implementation name."""
```
Comment (kohr-h) [inline conftest.py]: an

File: conftest.py [hunk 2]
```diff
@@ -94,20 +105,47 @@ def floating_dtype(request):
...
+
+    Returns
+    -------
+    name : `str`
+        Name of the ufunc.
+    n_in : `int`
+        Number of input values of the ufunc.
+    n_out : `int`
+        Number of output values of the ufunc.
+    doc : `str`
```
Comment (kohr-h) [inline conftest.py]: No ticks for `int` and `str`

File: conftest.py [hunk 3]
```diff
@@ -94,20 +105,47 @@ def floating_dtype(request):
...
 
 @pytest.fixture(scope="module", ids=reduction_ids, params=reduction_params)
 def reduction(request):
+    """Tuple with information on a reduction.
+
+    Returns
+    -------
+    name : `str`
+        Name of the reduction.
+    doc : `str`
```
Comment (kohr-h) [inline conftest.py]: same here

**Comment by kohr-h** [comment]
Update also `.travis.yml` and `setup.py` (find in files `py\.test`)

**Comment by adler-j** [comment]
Should be nothing in .travis.yml and setup.py already fixed

**Comment by adler-j** [comment]
Merge after CI

**Comment by kohr-h** [comment]
> Should be nothing in .travis.yml

There's a call to `py.test` which should be changed to `pytest`. Do that on master.

**Comment by adler-j** [comment]
py.test is still valid, not sure if they are scrapping it but sure will do.

**Pull request merged**